src/Symfony/Component/DependencyInjection/Alias.php

```diff
@@ -22,7 +22,12 @@ class Alias
      */
     public function __construct($id, $public = true)
     {
-        $this->id = strtolower($id);
+        if (!is_string($id)) {
```

Comment:
@nicolas-grekas just finished mine :).. probably related. Not sure i follow.. but ill dig in :)